src/screens/WalletType.js

```diff
@@ -70,6 +70,17 @@ class WalletType extends React.Component {
    */
   handleVersion = (event, arg) => {
     if (arg.success) {
+      // compare ledger version with our min version
+      const version = arg.data.slice(3, 6).join('.');
+      hathorLib.storage.setItem('ledger:version', version);
+      if (
+        helpers.cmpVersionString(version, LEDGER_MIN_VERSION) < 0 ||
+        helpers.cmpVersionString(version, LEDGER_MAX_VERSION) >= 0
```

Comment:
I'm thinking about this. Maybe we shouldn't do this check here. We still don't know if this wallet version will support the next ledger version. If we improve some ledger UX but the wallet continue working well, it will not support the ledger version?